docker/postgres/docker-entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "linkedevents" <<-EOSQL
+	CREATE EXTENSION postgis;
```

Comment:
CREATE EXTENSION postgis IF NOT EXISTS (and the same for hstore)
is a bit more robust.

Reply:
👍